web/src/components/DandiAppBar.vue

```diff
@@ -59,12 +59,24 @@
         </v-btn>
       </template>
       <v-list dense>
+        <v-list-item
+          @click="reloadAPIKey"
+        >
+          <v-list-item-action class="mr-2">
+            <v-btn icon>
+              <v-icon>mdi-reload</v-icon>
+            </v-btn>
+          </v-list-item-action>
+          <v-list-item-title>{{ apiKey }}</v-list-item-title>
```

Comment:
See my top-level comment about copying text.   Maybe there's some part of that you can fit in here, maybe not.